### PR TITLE
Added JavaScript Sample in CONTRIBUTING.md

### LINKED ISSUE #288: JavaScript sample missing in CONTRIBUTING.md
- [ ] Issue Title is in form `Algo/DS Name [Language]`.
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This Algo/DS is worth adding to this repository.
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.

### PR DESCRIPTION
**Pull request by Anshuman-Verma**
Fixes #288 

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

**Comment by Anshuman-Verma** [comment]
@singhpratyush please review again, do correct me if I've gone wrong somewhere.
Thanks

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -137,3 +137,23 @@ func main() {
     fmt.Println(sortedArray)
 }
 ```
+
+#### JavaScript
+
+```JavaScript
+function quickSort (arr) {
+    /*
```
Comment (singhpratyush) [inline CONTRIBUTING.md]: Use TAB indentation of size 2 throughout.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -137,3 +137,23 @@ func main() {
...
+    Your implementation here
+    */
+}
+
+function main () {
+    var input = [2,3,0,4];
+    quickSort(input);
+    for(var x in input) {
```
Comment (singhpratyush) [inline CONTRIBUTING.md]: Space after `for`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -137,3 +137,23 @@ func main() {
...
+
+```JavaScript
+function quickSort (arr) {
+    /*
+    Your implementation here
+    */
+}
+
+function main () {
+    var input = [2,3,0,4];
```
Comment (singhpratyush) [inline CONTRIBUTING.md]: `[2, 3, 0, 4];`

Comment (singhpratyush) [inline CONTRIBUTING.md]: Use `let` instead.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Anshuman-Verma) [inline CONTRIBUTING.md]: Don't get the point of TAB indentation of size 2 **THROUGHOUT**.
All the above samples have space indentation with zero throughout TAB indentation

**Comment by Anshuman-Verma** [comment]
@singhpratyush done the requested changes, switched to TAB spacing but didn't get the point of

> 2 TAB spacing throughout

Have a look if this is appropriate.

**Comment by aviaryan** [comment]
@Anshuman-Verma This looks ok now.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -137,3 +137,23 @@ func main() {
...
+function quickSort (arr) {
+	/*
+	Your implementation here
+	*/
+}
+
+function main () {
+	let input = [2, 3, 0, 4];
+	quickSort(input);
+    	for (let x in input) {
```
Comment (aviaryan) [inline CONTRIBUTING.md]: Spaces indentation here. Also please check other lines of this code.

**Comment by Anshuman-Verma** [comment]
@aviaryan @singhpratyush I think this  should do it.

**Comment by Anshuman-Verma** [comment]
Switched back to TAB indentation of 2

**Comment by Anshuman-Verma** [comment]
Finally ! :laughing:

**Pull request merged**